### PR TITLE
DIV-6637: allow share a case

### PR DESCRIPTION
**Pull request by yarpo**
According to:
https://tools.hmcts.net/confluence/display/ACA/API+Operation%3A+Assign+Access+within+Organisation

we need to handle it in post-commit callback, which means:

```
CallBackURLSubmittedEvent
```

### CODE REVIEW

**Review by jacmurphy — APPROVED** [review]

**Review by jvictorcf — COMMENTED** [review]
@yarpo, what's with the sub-module changes.
You might have to update the submodule.

**Comment by madjava** [comment]
Had an issue with @Dizrupto When he noticed changes to the `ccd-definition-processor` folder. Not 100% sure what is was but might want to check with him still before merge.
@Dizrupto FYI

**Review by madjava — APPROVED** [review]

**Review by Dizrupto — APPROVED** [review]

**Comment by Dizrupto** [comment]
@madjava, @jvictorcf 
was my bad last time actually it just seems ccd-definition-processor wasn't pointing to the latest version.
Patryk ran "yarn reset-ccd-submodule" so should be fine and it should be pointing to the latest ccd-definition-processor now :)

File: definitions/divorce/json/CaseEvent/CaseEvent.json [hunk 1]
```diff
@@ -2747,7 +2747,7 @@
     "PostConditionState": "SOTAgreementPayAndSubmitRequired",
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/solicitor-create",
     "CallBackURLAboutToStartEvent": "${CCD_DEF_COS_URL}/default-values",
-    "RetriesTimeoutURLAboutToSubmitEvent": "120,120",
+    "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/allow-share-a-case",
```
Comment (Dizrupto) [inline definitions/divorce/json/CaseEvent/CaseEvent.json:2750]: Just remembered, actually this needs to go under a toggle.
CaseEvent-share-a-case-nonprod.json

Comment (yarpo) [reply definitions/divorce/json/CaseEvent/CaseEvent.json]: yes, you're right. Done.

**Review by jvictorcf — APPROVED** [review]

**Pull request merged**